README.md

```diff
@@ -1,36 +1,56 @@
-This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
+# Lupus Decoupled Next.js Demo
 
-## Getting Started
+## What is supported?
+- Rendering custom elements via components
+- Rendering a tree of custom elements
+- Forwarding /some-page requests to the backend and rendering results
+- breadcrumbs
+- metatags
 
-First, run the development server:
+## Manual setup steps
```

Comment:
documenting local, manual setup is good, but more important is documentation on "Try it online" with gitpod. --> Provide a gitpod link that works with this repo so things can be tested with it easily.

Reply:
updated README.md file